### PR TITLE
Audit Cloud Databases FAQ: INFODEV-1378

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-25b8df77d1/how-to/)

File: content/cloud-databases/cloud-databases-faq.md [hunk 1]
```diff
@@ -232,30 +255,31 @@ Yes. A Cloud Databases support ticket category is available in the
...
-For more information about these engine types, see the MySQL
+For more information about these engine types, see the following MySQL
 documentation:
 
--   [InnoDB Storage Engine
-    documentation](http://dev.mysql.com/doc/refman/5.1/en/innodb-storage-engine.html)
--   [MyISAM Storage Engine
-    documentation](http://dev.mysql.com/doc/refman/5.1/en/myisam-storage-engine.html)
+-   [The InnoDB Storage
+    Engine](http://dev.mysql.com/doc/refman/5.1/en/innodb-storage-engine.html)
```
Comment (derekmoyes) [inline content/cloud-databases/cloud-databases-faq.md]: https://dev.mysql.com/doc/refman/5.7/en/innodb-storage-engine.html

File: content/cloud-databases/cloud-databases-faq.md [hunk 2]
```diff
@@ -232,30 +255,31 @@ Yes. A Cloud Databases support ticket category is available in the
...
+-   [The MyISAM Storage
+    Engine](http://dev.mysql.com/doc/refman/5.1/en/myisam-storage-engine.html)
```
Comment (derekmoyes) [inline content/cloud-databases/cloud-databases-faq.md]: https://dev.mysql.com/doc/refman/5.7/en/myisam-storage-engine.html

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-1ce6cd5c93/how-to/). :bow:

**Comment by RackerKate** [comment]
@cyrichardson For your review.

**Comment by cyrichardson** [comment]
@RackerKate  - Suggestions here:
[cloud-databases-faq-cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1944593/cloud-databases-faq-cyrEdits.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-afcaddb36f/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-09bf6c5151/how-to/)

**Comment by cyrichardson** [comment]
@RackerKate Merged! This addresses #docs channel question from Charlie.

**Pull request merged**